### ISSUE TITLE
Tray icon to control music

### ISSUE THREAD

**Issue by rafaelleru** [rafaelleru/torrent-player#31]

**Comment by niralpokal** [comment]
For this, do you want to implement it using the tray api? https://electron.atom.io/docs/api/tray/

**Comment by rafaelleru** [comment]
yes, I have planned to work on it today and tomorrow, do you want to do this, I have no problem and can do other stuff like write some unit test 😑 hahahaha

**Comment by niralpokal** [comment]
Haha you can work on this if you already planned it. I can write those tests.
